### ISSUE TITLE
Synonymous variants are scored higher

### ISSUE THREAD

**Issue by jerome-f** [kircherlab/CADD-scripts#58]
Hi,

I noticed that many synonymous variants are scored higher in the CADD 1.7. 
for instance "21_46272794_C_T" new PHRED_Score = 18.55, v1.6 PHRED_Score = 12.00
The variant is synonymous. (pulled from the prescored files)

**Comment by makirc** [comment]
Hi Jerome, 
Thank you for your message and pointing to an example where a synonymous variant has changed the score. CADD v1.7 has included new annotations and is based on new training of the model. It is expected that scores change. This synonymous variant is in a very constraint part of the genome and has some indication of a potential splicing effect. It might well be that it gets scored more highly because of that in the new version. If you have some specific concerns/questions please let us know. 
Best, Martin

**Comment by jerome-f** [comment]
@makirc I agree with you on there is higher conservation component to the variant and can impact codon usage efficiency and/or splicing; Just wanted make a note here that this was observed.
